run.bat

```diff
@@ -0,0 +1,2 @@
+python ./run.py
```

Comment:
This is a windows-exclusive script.
We can just simply run `python ./run.py` instead.

Reply:
Should remove it.